docker/docker-compose-dev.yml

```diff
@@ -3,19 +3,31 @@ version: '3'
 services:
   q_yaar_core:
     restart: always
-    build: ./../q_yaar_platform
+    build: 
+      context: ./../q_yaar_platform
+      dockerfile: ./docker/Dockerfile-dev
     ports:
       - "8000:8000"
     depends_on:
       - q_yaar_redisearch
       - q_yaar_db
     networks:
       - backend
-    env_file: ./../q_yaar_platform/.env
-    command: gunicorn q_yaar.wsgi:application -w 4 -b :8000
+    environment:
+      - IS_LOCAL=True
```

Comment:
@paragjnath I don't think it's a good idea to have a copy of env here. And all our env variables will be exposed in code as well.
Can we just load the .env file like it was before?

Reply:
Done, moved pg environment variables to .env also as it also exposed secreats